### PR TITLE
[WIP] Better Signing approach

### PR DESCRIPTION
**Pull request by Neil3079**

### CODE REVIEW

**Comment by Neil3079** [comment]
I need to try this out on a project but hopefully this should make our signing much easier

File: Fastfile [hunk 1]
```diff
@@ -117,30 +117,14 @@ def _build_ipa
 end
 
 def _build_with_gym
-  install_provisioning_profiles
```
Comment (Neil3079) [inline Fastfile:120]: It might be worth still doing this

File: Fastfile [hunk 2]
```diff
@@ -117,30 +117,14 @@ def _build_ipa
...
-  UI.message('Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined')
-  'enterprise'
+  update_project_provisioning
+  gym
 end
 
 def _update_team_id_if_necessary
   project_path = ENV['FL_PROJECT_SIGNING_PROJECT_PATH']
-  team_id = _get_team_id
+  team_id = ENV['FL_PROJECT_TEAM_ID']
```
Comment (zacoid55) [inline Fastfile:127]: If the team id env variable is empty by accident is there a backup way we could auto-detect it?

**Review by zacoid55 — DISMISSED** [review]

File: Fastfile (hunk 1 above)
Comment (KaneCheshire) [reply Fastfile:120]: Not 100% sure but I feel like this might be broken anyway

**Review by KaneCheshire — DISMISSED** [review]

File: Fastfile (hunk 2 above)
Comment (Neil3079) [reply Fastfile:127]: The way we were doing this before was to investigate the export options plist to see if it was there, this PR aims to remove the need for a developer to supply this

**Review by zacoid55 — APPROVED** [review]

**Review by zacoid55 — APPROVED** [review]

**Pull request closed without merging**